src/options/options.requests.ts

```diff
@@ -1,14 +1,17 @@
-import {
-  NetworkSelectionRequest,
-} from '../services/common-interfaces';
 import { BigNumber, Wallet } from 'ethers';
-import { ContractReceipt } from 'ethers';
+import { ContractReceipt, PopulatedTransaction } from 'ethers';
+
+export type TransactionBuildingResult = {
```

Comment:
(arguably this should be in more of a types file than a request/response definition file, but i am ok with this for now)